src/GrommetSwagger.js

```diff
@@ -37,19 +37,35 @@ export default class GrommetSwagger extends Component {
     }
   }
 
+  getOrigin = (htmlHost, configHost, configUrl) => {
+    const parser = document.createElement('a');
+    parser.href = configUrl;
+    const origin = htmlHost || configHost || configUrl.origin;
```

Comment:
I think `configUrl.origin` should be `parser.origin`.

Reply:
Nice catch, thanks